dist/views/Renderer.js

```diff
@@ -0,0 +1,19 @@
+class Renderer {
+  
+  renderTeam = (teamData) => {
+    const source = $('#player-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(teamData);
+    $('.main-container').empty().append(newHTML);
+  };
+
+  renderPlayerStats = (playerStats, playerId) => {
+    const source = $('#stats-template').html();
+    const template = Handlebars.compile(source);
+    const newHTML = template(playerStats);
+    $(`.player[data-id=${playerId}]`)
+      .find('.player-img')
+      .empty()
+      .append(newHTML);
+  };
+}
```

Comment:
nice! other way to do that is to send 'this' from the event-listener to this function and then find it's .player-img class (of that specific player)